### PR TITLE
Make the slugrunner use an init container to download the app code

### PR DESCRIPTION
**Pull request by aguerra**
So it doesn't mount the storage keys anymore.

### CODE REVIEW

File: pkg/server/spec/pod_test.go [hunk 1]
```diff
@@ -86,6 +99,13 @@ func TestNewBuilder(t *testing.T) {
 	if ps.ContainerLimits.Memory != expectedContainerLimits.Memory {
 		t.Errorf("expected %s, got %s", expectedContainerLimits.Memory, ps.ContainerLimits.Memory)
 	}
+
+	if len(ps.VolumeMounts) != 1 {
+		t.Errorf("expected %d, got %d", 1, len(ps.VolumeMounts))
```
Comment (drgarcia1986) [inline pkg/server/spec/pod_test.go]: 💅 `t.Errorf("expected 1, got %d", len(ps.VolumeMounts))`

File: pkg/server/k8s/converters_test.go [hunk 2]
```diff
@@ -103,6 +103,25 @@ func TestPodSpecVolumesToK8sVolumes(t *testing.T) {
 	}
 }
 
+func TestPodSpecEmptyDirVolumeToK8s(t *testing.T) {
+	vols := []*spec.Volume{
+		{Name: "Vol-Test", EmptyDir: true},
+	}
+	k8sVols := podSpecVolumesToK8sVolumes(vols)
+
+	for idx, vol := range vols {
```
Comment (yagonobre) [inline pkg/server/k8s/converters_test.go]: Maybe remove this for?

File: pkg/server/exec/exec.go [hunk 3]
```diff
@@ -56,10 +57,14 @@ func (ops *ExecOperations) RunCommand(ctx context.Context, user *database.User,
 		return nil, errChan
 	}
 
+	imgs := &spec.SlugImages{
+		Runner: ops.defaults.RunnerImage,
+		Store:  ops.defaults.StoreImage,
+	}
```
Comment (drgarcia1986) [inline pkg/server/exec/exec.go]: I'm a bit curious, you created this struct to avoid another argument on `spec` functions?

Comment (aguerra) [reply pkg/server/exec/exec.go]: Yep, the  argument list is gigantic.

File: pkg/server/k8s/converters_test.go (hunk 2 above)
Comment (aguerra) [reply pkg/server/k8s/converters_test.go]: In fact, the function is slice based (the slices are not used for test cases) so I guess is easier to use a loop for testing.

Comment (yagonobre) [reply pkg/server/k8s/converters_test.go]: Make sense!

**Comment by aguerra** [comment]
@yagonobre @drgarcia1986 Updated. Also put SLUG_DIR as protected env var.

File: pkg/server/exec/exec.go (hunk 3 above)
Comment (drgarcia1986) [reply pkg/server/exec/exec.go]: hm, ok 👍 for a next refactor maybe we could create a struct with all args, something like `options{}`

**Pull request merged**